[2025-02-11 11:00:59,049] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: grippe_ETL_pipeline.load_vaccins2024 manual__2025-02-11T11:00:46.370922+00:00 [queued]>
[2025-02-11 11:00:59,078] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: grippe_ETL_pipeline.load_vaccins2024 manual__2025-02-11T11:00:46.370922+00:00 [queued]>
[2025-02-11 11:00:59,084] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-02-11 11:00:59,084] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2025-02-11 11:00:59,085] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-02-11 11:00:59,118] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): load_vaccins2024> on 2025-02-11 11:00:46.370922+00:00
[2025-02-11 11:00:59,128] {standard_task_runner.py:52} INFO - Started process 343 to run task
[2025-02-11 11:00:59,136] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'grippe_ETL_pipeline', 'load_vaccins2024', 'manual__2025-02-11T11:00:46.370922+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Vaccin.py', '--cfg-path', '/tmp/tmpx5_67iox', '--error-file', '/tmp/tmpxj0ezde_']
[2025-02-11 11:00:59,138] {standard_task_runner.py:80} INFO - Job 50: Subtask load_vaccins2024
[2025-02-11 11:00:59,274] {task_command.py:370} INFO - Running <TaskInstance: grippe_ETL_pipeline.load_vaccins2024 manual__2025-02-11T11:00:46.370922+00:00 [running]> on host 976dfa0d1440
[2025-02-11 11:00:59,450] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=grippe_ETL_pipeline
AIRFLOW_CTX_TASK_ID=load_vaccins2024
AIRFLOW_CTX_EXECUTION_DATE=2025-02-11T11:00:46.370922+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-11T11:00:46.370922+00:00
[2025-02-11 11:01:00,308] {logging_mixin.py:115} INFO - ✅ Chargement réussi dans la table grippe_vaccins2024 (52 lignes)
[2025-02-11 11:01:00,309] {python.py:173} INFO - Done. Returned value was: None
[2025-02-11 11:01:00,362] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=grippe_ETL_pipeline, task_id=load_vaccins2024, execution_date=20250211T110046, start_date=20250211T110059, end_date=20250211T110100
[2025-02-11 11:01:00,430] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-02-11 11:01:00,508] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
